### PR TITLE
Bug 1974338: Update README.md with USE_JAVA_VERSION env var explanation

### PR DESCRIPTION
**Pull request by waveywaves**
`USE_JAVA_VERSION` env var allows the user to set the java version of the jnlp client running in the jenkins agent image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,7 @@ initialization by passing `-e VAR=VALUE` to the Docker run command.
...
 |  `JENKINS_UC_INSECURE`       | When your Jenkins Update Center repository is using a self-signed certificate with an unknown Certificate Authority, this variable allows one to bypass the repository's SSL certificate check. The variable applies to download of the plugin which may occur during Jenkins image build, if you build an extension of the jenkins image or if you run the Jenkins image and leverage one of the options to download additional plugins (use of s2i whith plugins.txt or use of `INSTALL_PLUGINS` environment variable. |
 | `MAVEN_MIRROR_URL` | Allows you to specify a [Maven mirror repository](https://maven.apache.org/guides/mini/guide-mirror-settings.html) in the form of `MAVEN_MIRROR_URL=https://nexus.mycompany.com/repository/maven-public`. The mirror repository is used by maven as an additional location for artifacts downloads. For more details on [how maven mirrors works](https://maven.apache.org/guides/mini/guide-mirror-settings.html), you can refer to maven documentation.
+| `USE_JAVA_VERSION`	|	Allows you to set the Java version used by the JNLP Client for the agent images. By default the value is set to `java-11`	|
```
Comment (adambkaplan) [inline README.md:174]: What are the allowed values? What value should a developer provide to use Java 8?

**Review by adambkaplan — CHANGES_REQUESTED** [review]

**Comment by gabemontero** [comment]
/retitle Bug: 1974338: Update README.md with USE_JAVA_VERSION env var explanation

**Comment by gabemontero** [comment]
/retitle Bug 1974338: Update README.md with USE_JAVA_VERSION env var explanation

**Comment by gabemontero** [comment]
/lgtm

@rolfedh - we'll want to update the product docs for this as well.  Can we drive that off of this BZ, where @waveywaves perhaps crafts an openshift/openshift-docs PR for you?

Or do you prefer some other form or bookkeeping  / execution?

**Comment by gabemontero** [comment]
Also @rolfedh of course if you have opinions on the specific wording from @waveywaves let us know.

**Comment by rolfedh** [comment]
> /lgtm
> 
> @rolfedh - we'll want to update the product docs for this as well. Can we drive that off of this BZ, where @waveywaves perhaps crafts an openshift/openshift-docs PR for you?
> 
> Or do you prefer some other form or bookkeeping / execution?

Hi @gabemontero. I've created https://issues.redhat.com/browse/RHDEVDOCS-3177 to track the doc work for this Jenkins task. @Preeticp Here's another Jenkins-related doc task we need to consider for our planning.

**Pull request merged**